app/controllers/openid_connect/logout_controller.rb

```diff
@@ -48,6 +55,13 @@ def delete
       if result.success? && redirect_uri
         handle_logout(result, redirect_uri)
       else
+        if result.extra[:integration_errors].present?
+          analytics.integration_errors_present(
+            **result.
+              to_h[:integration_errors].
+              merge({ event: :oidc_logout_submitted }),
```

Comment:
supernit for style! you can drop the optional curly braces
```suggestion
              merge(event: :oidc_logout_submitted),
```

Reply:
updated in https://github.com/18F/identity-idp/pull/11615/commits/d011de2a5217dddf83ecedde0a086c8efb1f7eea